srcpkgs/ansible-core/template

```diff
@@ -1,17 +1,17 @@
 # Template file for 'ansible-core'
 pkgname=ansible-core
-version=2.11.0
+version=2.11.2
 revision=1
 build_style=python3-module
 hostmakedepends="python3-setuptools"
 depends="${hostmakedepends} python3-cryptography python3-Jinja2 python3-paramiko
- python3-yaml python3-packaging"
+ python3-yaml python3-packaging python3-resolvelib"
```

Comment:
turns out, ansible-galaxy wasn't working without resolvelib, so it's been packaged and added as a dependency now